Makefile

```diff
@@ -235,33 +235,13 @@ init_envoy: copy_envoy_config copy_gateway_endpoints ## Runs copy_envoy_config a
 
```

Comment:
@commoddity Can you please update the suffixes on endpoints 1,2,3 in `envoy/auth_server/endpoint_store/endpoint_store_test.go`?

Reply:
Done; updated the test data to be consistent with the Gateway Endpoints example file in PADS.